metashare/api/gh.py

```diff
@@ -0,0 +1,17 @@
+"""
+GitHub utilities
+"""
+
+from django.core.exceptions import MultipleObjectsReturned, ObjectDoesNotExist
+from github3 import login
+
+
+def get_all_org_repos(user):
+    try:
+        token = (
+            user.socialaccount_set.get(provider="github").socialtoken_set.get().token
+        )
+    except (ObjectDoesNotExist, MultipleObjectsReturned):
+        return []
+    gh = login(token=token)
+    return [repo.url for repo in gh.repositories()]
```

Comment:
@wlonk As far as I can tell, `repositories()` only returns repos owned by individuals, not by orgs. At least in my case I'm only seeing repos owned by myself or other individuals -- no OddBird or SFDO or other org repos. Do we need to combine this with `organizations()` and `org.repositories()` or something?

https://github3py.readthedocs.io/en/master/api-reference/github.html#github3.github.GitHub.organizations
https://github3py.readthedocs.io/en/master/api-reference/orgs.html#github3.orgs.ShortOrganization.repositories

Reply:
This is what I was saying in standup, that the results looked wrong; I saw some repos from orgs I'm part of, but no Oddbird ones, so I wasn't sure what was going on. I'll dig through those docs again and see.